Instruments/python/merge_trees.py

```diff
@@ -0,0 +1,42 @@
+# Merge multiple TTrees.
+# This file is part of https://github.com/hh-italian-group/AnalysisTools.
+import uproot 
+import argparse
+  
+parser = argparse.ArgumentParser(description='Merging Multiple TTrees.')
+parser.add_argument('--files', required=True, type=str, nargs='+', help="Input the files names")  
+parser.add_argument('--out', required=True, type=str, help="Output ROOT file")
+parser.add_argument('--tree', required=True, type=str, help="Tree name")    
+parser.add_argument('--tree-out', required=False, type=str, default=None, help=" out tree name") 
+parser.add_argument('--chunk-size', required=False, type=int, default=1000000, help="Number of entries per iteration")
+parser.add_argument('--compression_type', required=False, default='LZMA')
+parser.add_argument('--compression_level', required=False, type=int, default=9)
```

Comment:
Try to avoid mixing of the argument naming notation: or use `_` to separate words or `-`.